Content.Server/Explosion/Components/OnTrigger/SoundOnTriggerComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Server.Explosion.EntitySystems;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Explosion.Components;
+
+/// <summary>
+/// Will play sound from the attached entity upon a <see cref="TriggerEvent"/>.
+/// </summary>
+[RegisterComponent]
+public sealed class SoundOnTriggerComponent : Component
+{
+    [DataField("removeOnTrigger")]
+    public bool RemoveOnTrigger = true;
+
```

Comment:
Also this.

Reply:
https://github.com/space-wizards/space-station-14/pull/13747#discussion_r1257262732